test/test.cpp

```diff
@@ -3,10 +3,139 @@ bundle config --local path vendor/bundle
 bundle install
 bundle exec arduino_ci_remote.rb  --skip-compilation
  */
+#include <cassert>
+#include <vector>
 #include "Arduino.h"
 #include "ArduinoUnitTests.h"
 #include "Keypad.h"
+#include "ci/ObservableDataStream.h"
```

Comment:
If you format the document using clang-format then it might change the order and spacing of includes.